skyflow/vault/_client.py

```diff
@@ -65,6 +77,12 @@ def insert(self, records: dict, options: InsertOptions = InsertOptions()):
         return result
 
     def detokenize(self, records):
+        '''
+        Returns values that correspond to the specified tokens.
+
+        :param records: Takes a dictionary that contains the records key that takes an array of records to fetch from the vault.
```

Comment:
Dictionary that contains the \`records\` key, which is an array of records to fetch from the vault.